packages/expect/src/jest_matchers_object.js

```diff
@@ -35,6 +35,13 @@ export const setState = (state: Object) => {
 
 export const getMatchers = () => global[JEST_MATCHERS_OBJECT].matchers;
 
-export const setMatchers = (matchers: MatchersObject) => {
+export const setMatchers = (matchers: MatchersObject, isInternal: boolean) => {
+  for (const key in matchers) {
```

Comment:
In the spirit of https://github.com/babel/babel/pull/6748 can we change it to `forEach`?

Reply:
Interesting. Sure thing!